First2.0.Infra/Mappings/HistoricoMapping.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Fisrt2._0.Domain.Entidades;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace First2._0.Infra.Mappings
+{
+    public class HistoricoMapping
+        : IEntityTypeConfiguration<Historico>
+    {
+        public void Configure(EntityTypeBuilder<Historico> builder)
+        {
+            builder.Property(d => d.Descricao).HasColumnName(nameof(Historico.Descricao)).IsRequired();
```

Comment:
Pode remover o HasColumnName
```suggestion
            builder.Property(d => d.Descricao).IsRequired();
```